scripts/runtime-upgrade/test_runtime_upgrade.sh

```diff
@@ -1,7 +1,39 @@
+#!/usr/bin/env bash
 
-block=$1
+set -euxo pipefail
 
-./target/release/zeitgeist export-state $block --chain battery_park --pruning archive > test-upgrade.json
-node scripts/runtime-upgrade/adjustTestUpgradeJson.js
-./target/release/zeitgeist purge-chain --chain ./test-upgrade.json -y
-./target/release/zeitgeist --chain ./test-upgrade.json --alice
\ No newline at end of file
+VALIDATOR_KEY="0x57f8dc2f5ab09467896f47300f0424385e0621c4869aa60c02be9adcc98a0d1d"
+ALICE_VALIDATOR="0x04d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d"
+
+SUDO_KEY="0x5c0d1176a568c1f92944340dbfed9e9c530ebca703c85910e7164cb7d1c9e47b"
+ALICE_SUDO="0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d"
+
+BLOCK=$1
+ID=battery_park_runtime_upgrade_test
+PRODUCTION_IMAGE=zeitgeistpm/zeitgeist-node:latest
+
+# Sync with live network
+#
+# Comment this whole section if `/tmp/migration` is already synced.
+
+sudo docker run -d --name migration -v /tmp/migration:/zeitgeist/data $PRODUCTION_IMAGE --base-path /zeitgeist/data --chain battery_park --pruning archive
```

Comment:
After trying different types of combinations for days, the only way I could find to bind host's `/tmp/migration` to container's `/zeitgeist/data` was using `--privileged`.

Even with the usage of the same UID/GIG along side the correct folder/files permissions, Substrate will throw `Permission Denied`